telegraf/telegraf.base

```diff
@@ -0,0 +1,131 @@
+# Telegraf Configuration
+#
+# Configuration for telegraf agent
+[agent]
+  interval = "10s"
+  round_interval = true
+  flush_interval = "10s"
+  metric_batch_size = 1000
+  metric_buffer_limit = 10000
+
+###############################################################################
+#                            OUTPUT PLUGINS                                   #
+###############################################################################
+# Define the main metric output file, excluding aggregated stats and 
+# Performance Playbook (estat) data.
+[[outputs.file]]
+  files = ["/var/delphix/server/log/stats/metrics.json"]
```

Comment:
Should we consider putting the data somewhere other than `/var/delphix`? It seems like we're creating a generic monitoring platform that will be useful even if `/var/delphix` doesn't exist. Maybe `/var/log/telegraf` would be better?

Reply:
The path includes it by default in support bundles, and runs with the mgmt stack as per the delphix-stat service (so I'm assuming that means /var/delphix would be there). I have no preference if you want to pull it out somewhere else though, as long as it's pulled into a bundle.

Reply:
I changed this to /var/log/telegraf, works fine, also verified it gets sucked into a support bundle. Done.